x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/data_tier_allocation_field/components/will_use_fallback_tier_notice.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React, { FunctionComponent } from 'react';
+import { EuiCallOut } from '@elastic/eui';
+
+import { PhaseWithAllocation, DataTierRole } from '../../../../../../../../../common/types';
+
+const nodeRoleToFallbackTierMap: Partial<Record<DataTierRole, string>> = {
+  data_hot: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.dataTierHotLabel', {
+    defaultMessage: 'hot',
+  }),
+  data_warm: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.dataTierWarmLabel', {
+    defaultMessage: 'warm',
+  }),
+};
+
+const i18nTexts = {
+  warm: {
+    title: i18n.translate('xpack.indexLifecycleMgmt.warmPhase.dataTier.willUseFallbackTierTitle', {
+      defaultMessage: 'No nodes assigned to the warm tier',
+    }),
+    body: (nodeRole: DataTierRole) =>
+      i18n.translate('xpack.indexLifecycleMgmt.warmPhase.dataTier.willUseFallbackTierDescription', {
+        defaultMessage:
+          'This policy will move data in the warm phase to {tier} tier nodes instead.',
```

Comment:
I think _move_ is a little misleading, as the net effect is likely to be that it isn't moved to a different tier, it stays where it is.

```suggestion
          If no warm nodes are available, data is store in the {tier} tier.',
```

Reply:
Great point!

Reply:
With this change the callout will state:
> **No nodes assigned to the warm tier**
> If no warm nodes are available, data is stored in the hot tier.

I wonder if this sounds a little unclear. The title is definitive about no warm nodes being available, and then the copy seems to walk it back by cautioning "if" they're unavailable. But there's no "if" involved -- we know they're unavailable, so maybe we should more directly explain the consequence?